layers/meta-balena-fsl-arm/recipes-bsp/u-boot/u-boot-solidsense-imx6_%.bbappend

```diff
@@ -0,0 +1,6 @@
+inherit resin-u-boot
```

Comment:
You should also define UBOOT_KCONFIG_SUPPORT: https://github.com/balena-os/balena-rockpi/blob/master/layers/meta-balena-rockpi/recipes-bsp/u-boot/u-boot-rockpi-4.bbappend#L3-L5